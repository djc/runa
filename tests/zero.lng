def main(name: str, args: array[str]):
	print(0)
	print(1 + 0)
	print(0 - 1)
	print('a' + '')
