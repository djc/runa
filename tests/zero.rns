def main(name: &str, args: &array[str]) -> i32:
	print(0)
	print(1 + 0)
	print(0 - 1)
	print('a' + '')
	return 0
