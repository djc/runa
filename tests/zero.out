0
1
-1
a
